bootstrap/src/lib/cache/index.ts

```diff
@@ -9,19 +9,28 @@ import { RedisOptions } from './redis'
 const DEFAULT_CACHE_TYPE = 'local'
 const DEFAULT_CACHE_KEY_GROUP = uuid()
 const DEFAULT_CACHE_KEY_IGNORED_PROPS = ['id', 'maxAge']
+const DEFAULT_CACHE_RATE_CAPACITY = '1000000'
+const DEFAULT_CACHE_KEY_RATE_LIMIT_PARTICIPANT = uuid()
+
 // Request coalescing
 const DEFAULT_RC_INTERVAL = 100
 const DEFAULT_RC_INTERVAL_MAX = 1000
 const DEFAULT_RC_INTERVAL_COEFFICIENT = 2
 const DEFAULT_RC_ENTROPY_MAX = 0
 
+const DEFAULT_RATE_WEIGHT = 1
+const DEFAULT_RATE_COST = 1
+
 const env = process.env
 export const defaultOptions = () => ({
   enabled: parseBool(env.CACHE_ENABLED),
   cacheOptions: defaultCacheOptions(),
   cacheBuilder: defaultCacheBuilder(),
   key: {
     group: env.CACHE_KEY_GROUP || DEFAULT_CACHE_KEY_GROUP,
+    rateLimitGroup: env.API_KEY || undefined,
+    rateLimitParticipant: DEFAULT_CACHE_KEY_RATE_LIMIT_PARTICIPANT,
+    totalCapacity: parseInt(env.CACHE_RATE_CAPACITY || DEFAULT_CACHE_RATE_CAPACITY),
```

Comment:
What is capacity exactly? Rate limit per second, hour or day? This should affect to the maxAge calculation